content/en/providers/operations-manuals/howto01_using_igtf_ca_distribution/_index.md

```diff
@@ -0,0 +1,322 @@
+
+For WLCG sites that are migrating from the lcg-CA package: the WLCG policy
+companion of the EGI templates can be found at
+[QWG](http://lcg-ca.web.cern.ch/lcg-ca/distribution/current/meta/ca-policy-lcg.tpl)
+and
+[CDB](http://lcg-ca.web.cern.ch/lcg-ca/distribution/current/meta/pro_software_meta_ca_policy_lcg.tpl)
+and can be included in the profile in parallel with the EGI core template. All
+packages needed are also included in the EGI repository, so only a single
+repository reference is necessary.
+
+## Trust Anchor structure for differentiated (collaborative) assurance
+
+EGI is moving towards support for 'differentiated' or collaborative assurance,
+where different elements of trust are provided by several collaborating parties:
+the identity providers, user home organisations, and research communities. This
+means that trust anchors with varying trust assurance profiles will be
+distributed for use in distinct EGI usage scenarios. Starting mid-2017 (release
+1.82 or above) initially two profiles will be supported. For a discussion on the
+impact, see [EGI document 2745](https://documents.egi.eu/document/2745).
+
+### Notice for sites in EGI that support WLCG
+
+The WLCG collaboration - having so requested EGI and in line with long-term EGI
+developments in AAI - has included a specific CA to its own trust anchor
+distribution. Sites that are also part of WLCG should install BOTH the
+"egi-core" AND "lcg" meta-packages, according to your policies. Note that your
+organisation or NGI may also have a specific policy and may have added or
+removed CAs compared to the EGI core policy. Sites that need compliance with the
+WLCG policy should install BOTH packages, or you will miss out the CERN WLCG
+IOTA CA specific exception see
+[EGI document: Considerations on the coexistence of controlled and flexible community models](https://documents.egi.eu/document/2745)
+for details and the
+[WLCG statement](http://lcg-ca.web.cern.ch/lcg-ca/doc/WLCG-CERN-IOTA-statement-MB.pdf).
+
+The EGI production repository contains all relevant packages for both policies
+(and the WLCG repository contains also all EGI packages). There is no reason to
+reconfigure repository locations. Specifically installing (adding) also the
+`ca-policy-lcg` package may be necessary for system configurations built after
+May 2009.
+
+### Combined Assurance/Adequacy Model
+
+In the first quarter of 2017, full support for differentiated assurance profiles
+(combined assurance/adequacy model, also called "cam") will be introduced in the
+EGI trust fabric infrastructure. This will take the form of an additional trust
+anchor meta-package, and replaces the specific policy mechanism described above.
+Such full support also required _new software and configuration at each resource
+centre_. You must deploy the additional trust anchor meta-packages and the new
+local policies in unison, and never install the _cam_ package without such
+software support.
```

Comment:
dlgroep, please suggest a replacement/updated.

Reply:
The release notes (that have come with every version for the past years), actually has the better text for this subject:

"
The release contains a "cam" (combined assurance/adequacy) package based on the approved policy on differentiated assurance. Technically, this means that you must ONLY install the new ca-policy-egi-cam packages if you ALSO at the same time implement VO-specific authorization controls in your software stack. This may require reconfiguration or a software update. 
Otherwise, just only install or update the regular ca-policy-egi-core package. There are no changes in this case. The ca-policy-egi-core package is approved for all VOs membership and assurance models. No configuration change is needed.

Acceptable Authentication Assurance 
-----------------------------------------
If a VO registration service or e-Infrastructure registration service is accredited by EGI to meet the approved authentication assurance, an IGTF "DOGWOOD" accredited Authority - used solely in combination with said registration service - is also adequate for user authentication. See the policy at https://documents.egi.eu/document/2930 for details.
This additional restriction policy must be implemented by each service in the authorization software. The "combined assurance" model package MUST NOT be installed unless the additional authorization is in place. You will need to reconfigure and may need to install upgrades.
Not installing the new "cam" package does not have any detrimental effect on
current users - only a new class of users (that can only obtain an opaque
identifiers and do not do full vetting at their electronic identity provider)
could be affected, and then only those users that are member of one of the
communities that has part of the combined-assurance programme: LCG-Atlas,
LCG-Alice, LCG-LHCb, and LCG-CMS.
"

Reply:
Review comments in-line in the code

Reply:
Thanks, implemented.